spring-cloud-gcp-docs/src/main/asciidoc/cloudfoundry.adoc

```diff
@@ -1,15 +1,11 @@
 == Cloud Foundry
 
-Spring Cloud GCP provides support for Cloud Foundry's
-https://docs.pivotal.io/partners/gcp-sb/index.html[GCP Service Broker].
-Our Pub/Sub, Spanner, Storage and Stackdriver Trace starters are Cloud Foundry aware and
-retrieve properties like project ID, credentials, etc., that are used in auto configuration.
-
-In cases like Pub/Sub's topic and subscription, or Storage's bucket name, where those parameters
-are not used in auto configuration, you can fetch them using the VCAP mapping provided by Spring
-Boot.
-For example, to retrieve the provisioned Pub/Sub topic, you can use the
-`vcap.services.mypubsub.credentials.topic_name` property from the application environment.
-
-NOTE: If the same service is bound to the same application more than once, the auto configuration
-will not be able to choose among bindings and will not be activated for that service.
+Spring Cloud GCP provides support for Cloud Foundry's https://docs.pivotal.io/partners/gcp-sb/index.html[GCP Service Broker].
+Our Pub/Sub, Spanner, Storage, Stackdriver Trace and Cloud SQL MySQL and PostgreSQL starters are Cloud Foundry aware and retrieve properties like project ID, credentials, etc., that are used in auto configuration from the Cloud Foundry environment.
+
+In cases like Pub/Sub's topic and subscription, or Storage's bucket name, where those parameters are not used in auto configuration, you can fetch them using the VCAP mapping provided by Spring Boot.
+For example, to retrieve the provisioned Pub/Sub topic, you can use the `vcap.services.mypubsub.credentials.topic_name` property from the application environment.
+
+NOTE: If the same service is bound to the same application more than once, the auto configuration will not be able to choose among bindings and will not be activated for that service.
+
+NOTE: If Cloud SQL MySQL and PostgreSQL are simultaneously enabled in your Cloud Foundry app, the PostgreSQL settings will have precedence over the MySQL ones.
```

Comment:
Why can't we reject such a configuration as well? The same way like you said in the previous sentence?
Thanks